notebooks/demo.ipynb

```diff
@@ -33,38 +33,46 @@
    "execution_count": 2,
    "id": "8ef3a29d",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "application/vnd.jupyter.widget-view+json": {
-       "model_id": "0299875a115143f7abff5cc21e553c4b",
-       "version_major": 2,
-       "version_minor": 0
-      },
-      "text/plain": [
-       "UnfoldedMap(mapUUID='ae4f5345-8507-49ec-85f6-a1f8c7bc2219')"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "my_map = UnfoldedMap(mapUUID='ae4f5345-8507-49ec-85f6-a1f8c7bc2219')\n",
-    "my_map"
+    "my_map = UnfoldedMap(mapUUID='ae4f5345-8507-49ec-85f6-a1f8c7bc2219')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "8768bd3f",
+   "id": "3e50723f-9853-4d8f-a61d-d1e3b651c1b9",
    "metadata": {},
    "source": [
-    "After waiting for the map to finish loading, we can issue commands to the map:"
+    "We can use the [`jupyterlab-sidecar`](https://github.com/jupyter-widgets/jupyterlab-sidecar) package to display the map in JupyterLab's sidebar."
```

Comment:
Small nit: here I'd add "You can install the package with `pip install sidecar`".

In the Binder image, `sidecar` is preinstalled as a dependency, but if a user tries to run it on their own computer, they might not have `sidecar` already.